Bottom: 5d5c0a05d9e55f3c0d935ba9279d96b29ade67db
Top:    c8803d1a87f4572e8f76167bdb0f28e1be016b9e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 02:28:49 +0100

Refresh of add-support-for-a-poured-objec-I

---

diff --git a/src/draw.c b/src/draw.c
index 1e76156..9ce08b1 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -970,6 +970,16 @@ pour_callback (const BoxType * b, void *cl)
   struct pour_info *i = (struct pour_info *) cl;
   PourType *pour = (PourType *)b;
 
+  if (TEST_FLAG (SELECTEDFLAG | FOUNDFLAG, pour))
+    {
+      if (TEST_FLAG (SELECTEDFLAG, pour))
+        gui->set_color (Output.fgGC, i->Layer->SelectedColor);
+      else
+        gui->set_color (Output.fgGC, PCB->ConnectedColor);
+    }
+  else
+    gui->set_color (Output.fgGC, i->Layer->Color);
+
   if (gui->gui)
     _draw_pour (pour);
